### PR TITLE
[FIL-251] Add google analytics

### PR DESCRIPTION
**Pull request by Filip-L**

### CODE REVIEW

File: .github/workflows/build-docker-image.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ jobs:
           echo "NEXT_PUBLIC_DMOB_API_URL='${{ vars.NEXT_PUBLIC_DMOB_API_URL }}'" >> .env
           echo "NEXT_PUBLIC_GLIF_URL='${{ vars.NEXT_PUBLIC_GLIF_URL }}'" >> .env
           echo "NEXT_PUBLIC_IS_TESTNET='${{ vars.NEXT_PUBLIC_IS_TESTNET }}'" >> .env
+          echo "NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID='${{ vars.NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID }}'" >> .env
```
Comment (kacperzuk-neti) [inline .github/workflows/build-docker-image.yml:68]: Did we already add it to repo vars? If not, let's do it even before we merge.

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]
We need a cookie notice and GA should be conditional on accepting them. Additionally we should make sure we don't collect any PII.
https://lifesight.io/how-to/google-analytics-gdpr-compliant/ seems like a good guideline.

File: cos.json
Comment (kacperzuk-neti) [inline cos.json:1]: What's that?

**Review by tplocic20 — APPROVED** [review]

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]
Still missing new `NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID` variable in repo settings for staging and prod envs.
Please add it first before we merge.

File: .github/workflows/build-docker-image.yml (hunk 1 above)
Comment (lukasz-wal) [reply .github/workflows/build-docker-image.yml:68]: done

**Review by kacperzuk-neti — APPROVED** [review]

**Pull request merged**